tests/test_idempotency.py

```
@@ -0,0 +1,241 @@
+            registrar=registrar,
+            slug_hint="test-conv",  # Control the slug
+        )
+
+        assert not result.skipped
+        assert "New content from remote" in md_path.read_text()
+
+
+class TestSyncIdempotency:
+    """Test sync operations are idempotent."""
+
+    def test_sync_twice_produces_same_result(self, tmp_path):
```

Ruff (0.15.9)

[warning] Unused method argument: `tmp_path`

(ARG002)